README.md

```diff
@@ -61,7 +61,7 @@ need [`Node.js`](https://nodejs.org/en/) and [`git`](https://git-scm.com/).
 
 After you have the dependencies, the first-time setup is as follows:
 
-1. checkout this repository —
+1. check out this repository —
```

Comment:
both forms checkout vs check out are grammatically correct, however in this context we should be using checkout

Reply:
[Well](https://www.grammarphobia.com/blog/2019/05/checkout.html)...
>If the verb is intended, then the screen should read “Check Out,”  as if the instruction were short for “Check Out Here.”
>
>If the adjective is intended, the screen should read “Checkout”—as if short for “Checkout Option.”
>
>And if the noun is intended, the screen should also read “Checkout”—as if short for “Book Checkout.”

[Also](http://www.enhancemywriting.com/checkout-vs-check-out) --
> Checkout is a noun, and like most other nouns, it is only a single word. Check out is a phrasal verb, and like most other phrasal verbs it is two words.

Just to mess with us (as with many commands on Unix-like systems, which do not behave according to English rules), the `git` *command* is **`checkout`** -- which you use to **check out** the code.